Gemfile

```diff
@@ -58,7 +58,8 @@ group :development do
   gem 'guard-rspec'
   gem 'rb-fsevent', '~> 0.9.1'
   gem 'unicorn-rails'
-  gem 'rubocop'
+  gem 'traceroute'
+  # gem 'rubocop'
```

Comment:
Consider adding `require: false` instead of commenting out the line.

`gem 'rubocop', require: false`